src/cascadia/TerminalSettingsEditor/ColorSchemeViewModel.cpp

```diff
@@ -74,6 +74,11 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         _NotifyChanges(L"IsDefaultScheme");
     }
 
+    bool ColorSchemeViewModel::IsInBoxScheme() const
+    {
+        return _scheme.Origin() != Model::OriginTag::User;

```

Comment:
nit: the function name and body are a little confusing since `OriginTag::InBox` exists, so from the function name I'd expect the body to just be `_scheme.Origin() == OriginTag::InBox`. Maybe rename the function to `IsNotUserScheme`?

Reply:
Good point. I flipped the whole thing around to be `IsEditable`